client/py_client/helpers.py

```diff
@@ -25,3 +26,8 @@ def read_cid():
     cid = f.read()
     f.close()
     return cid
+
+
+def mkdir_p(path):
+    """ Surprisingly, there is no simple function in python to create a dir if it does not exist."""
```

Comment:
https://stackoverflow.com/questions/273192/how-can-i-safely-create-a-nested-directory-in-python
from pathlib import Path
Path("/my/directory").mkdir(parents=True, exist_ok=True)